File: async-scala-client/src/main/scala/io/swagger/client/model/SalesTransactionOut.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class SalesTransactionOut (
  header: SalesHeaderOut,
lines: List[SalesLinesOut],
calculatedTaxSummary: SalesCalculatedTaxSummaryForService,
processingInfo: ProcessingInfo
)

File: javascript-closure-angular-client/API/Client/SalesTransactionOut.js
goog.provide('API.Client.SalesTransactionOut');

/**
 * @record
 */
API.Client.SalesTransactionOut = function() {}

/**
 * @type {!API.Client.SalesHeaderOut}
 * @export
 */
API.Client.SalesTransactionOut.prototype.header;

/**
 * @type {!Array<!API.Client.SalesLinesOut>}
 * @export
 */
API.Client.SalesTransactionOut.prototype.lines;

/**
 * @type {!API.Client.SalesCalculatedTaxSummaryForService}
 * @export
 */
API.Client.SalesTransactionOut.prototype.calculatedTaxSummary;

/**
 * @type {!API.Client.ProcessingInfo}
 * @export
 */
API.Client.SalesTransactionOut.prototype.processingInfo;

